app/models/foreman_tasks/task/dynflow_task.rb

```diff
@@ -97,16 +103,38 @@ def cli_example
 
     def main_action
```

Comment:
This method sometimes returns an `Action` and sometimes the class of the active job. In either case we try to call `caller_execution_plan_id` on it here https://github.com/theforeman/foreman-tasks/pull/238/files#diff-a00196c0418a614248a22c431ef0d163R17 . Either all our ActiveJobs should inherit from a single class which would basically just stub this method and return `nil`, or preferably make the `main_action` always return the action and create another method for obtaining the job's class